docs/table/pandas.rst

```diff
@@ -53,17 +53,124 @@ It is also possible to create a table from a :class:`~pandas.DataFrame`::
 
 .. EXAMPLE END
 
+Details
+-------
 The conversions to and from ``pandas`` are subject to the following caveats:
 
 * The :class:`~pandas.DataFrame` structure does not support multidimensional
   columns, so |Table| objects with multidimensional columns cannot be converted
   to :class:`~pandas.DataFrame`.
 
-* Masked tables can be converted, but in columns of ``float`` or string values
-  the resulting :class:`~pandas.DataFrame` uses `numpy.nan` to indicate missing
-  values. For ``float`` columns, the conversion therefore does not necessarily
-  round-trip if converting back to an ``astropy`` table, because the
-  distinction between `numpy.nan` and masked values is lost. This is not a
-  problem for integer columns.
+* Masked tables can be converted, but pandas uses a sentinel such as `numpy.nan` to
+  represent missing values. Astropy uses a separate mask array to represent masked
+  values, where the value "under the mask" is preserved. When converting any astropy
+  masked column to a pandas DataFrame, the original values are lost.
+
+* Tables with :ref:`mixin_columns` such as `~astropy.time.Time`,
+  `~astropy.coordinates.SkyCoord`, and `~astropy.units.Quantity` can be converted, but
```

Comment:
These two forms are interchangable right? I see one form here and one form there; should we just pick one and stick with it?
```suggestion
  `~astropy.coordinates.SkyCoord`, and |Quantity| can be converted, but
```